.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous Build
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+
+jobs:
+    build-test:
+        runs-on: windows-latest
+
+        env:
+            DOTNET_MULTILEVEL_LOOKUP: 1
+
+        strategy:
+            fail-fast: false
+            matrix:
+                version: [net452, netcoreapp2.0, netcoreapp3.1]
```

Comment:
Any reason we're targeting netcoreapp2.0 out of support?  Why not target 2.1

```suggestion
                version: [net452, netcoreapp2.1, netcoreapp3.1]
```

Reply:
Never mind for now, I see that the test projects are currently referencing netcoreapp2.0.  Much better to get CI up and running any make those changes in another pr.

Reply:
Yes. We should update our targeted frameworks to the supported ones for the whole SDK some time soon.

Reply:
I've got a PR out there to do just that.  I'll update it once we have this merged.